src/App.tsx

```
@@ -5,7 +5,7 @@ import './App.css'
 
 function App() {
   const [count, setCount] = useState(0)
-
+  const [name, setName] = useState('')
```

ESLint

[error] 'name' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)